source/documentation/deploying_services/postgresql.md

```diff
@@ -271,6 +271,93 @@ cf update-service SERVICE_NAME -c '{"reboot": true, "force_failover": true}'
 
 When you force a failover, your PostgreSQL database IP address will change. The database's hostname will not change. You must configure your app to close all database connections to the previous IP address after forcing a failover.
 
+### Add or remove extensions for a Postgresql service instance
+
+GOV.UK PaaS supports and enables the following extensions by default:
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+|Extension|PostgreSQL version|
+|:---|:---|
+|uuid-ossp|9.5, 10|
+|postgis|9.5, 10|
+|citext|10|
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+GOV.UK PaaS supports the following extensions, but they are not enabled by default:
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+|Extension|PostgreSQL version|
+|:---|:---|
+|pg_stat_statements|9.5, 10|
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
+
+You can add or remove extensions for a PostgreSQL service instance by:
+
+- creating a new service instance
+- updating an existing service instance
+- restoring a PostgreSQL service instance snapshot
+
+#### Create a new service instance
+
+You [create a new PostgreSQL service instance](#set-up-a-postgresql-service) by running `cf create-service`. You can enable extensions in this new service instance by running:
+
+```
+cf create-service SERVICE_NAME -c '{"enabled_extensions": ["EXTENSION_1","EXTENSION_2",...,"EXTENSION_N"]}'
+```
+
+where:
+
+- `SERVICE_NAME` is a unique descriptive name for this service instance
+- `EXTENSION_1...N` are the names of the extensions you want to enable
+
+#### Update an existing service instance
+
+You add or remove extensions on an existing PostgreSQL service instance by updating your service with the new list of extensions:
```

Comment:
We should leave out removing extensions completely and only add it when we really plan to support removal. (This might depend partly on the outcome of the update story)

Reply:
keeping in with `diable_extensions`